x-pack/test/functional/apps/saved_objects_management/spaces_integration.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+const getSpacePrefix = (spaceId: string) => {
+  return spaceId && spaceId !== 'default' ? `/s/${spaceId}` : ``;
+};
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const testSubjects = getService('testSubjects');
+  const PageObjects = getPageObjects([
+    'common',
+    'security',
+    'savedObjects',
+    'spaceSelector',
+    'settings',
+  ]);
+
+  const spaceId = 'space_1';
+
+  describe('TOTO spaces integration', () => {
```

Comment:
😄

Reply:
This is my hidden keyword for `--grep` since my kid typed it one year ago 😄